_schemaV2.graphql

```diff
@@ -8106,13 +8106,22 @@ type Partner implements Node {
   # A connection of artworks from a Partner.
   artworksConnection(
     after: String
+
+    # Return only artwork(s) included in this list of IDs.
+    artworkID: [String]
     before: String
     exclude: [String]
     first: Int
     forSale: Boolean
+
+    # Return artworks with less than x additional_images.
+    imageCountLessThan: Int
     last: Int
+
+    # Return artworks published less than x seconds ago.
+    publishedWithin: Int
     sort: ArtworkSorts
-  ): ArtworkConnection @deprecated(reason: "Use `filterArtworksConnection`")
```

Comment:
Nice!

Reply:
🤩 

Cool! Ive never un-deprecated a field before in graphql, it is just as easy as removing the `@deprecated` decorator ? Neat

Reply:
yeah, the `@deprecated` decorator just means "you shouldn't use this" and doesn't really enforce that in any way, so should be as simple as this